packages/fast-usdc/package.json

```diff
@@ -37,6 +37,7 @@
     "@agoric/notifier": "^0.6.2",
     "@agoric/orchestration": "^0.1.0",
     "@agoric/store": "^0.9.2",
+    "@agoric/vat-data": "^0.5.2",
```

Comment:
This seems to be used for `makeScalarBigMapStore`. We're already using the zone api; it seems more straightforward to use that.